### DISCUSSION TITLE
Translatable interface

### DISCUSSION THREAD

**Discussion by hugoroy** [tosdr/tosdr.org#58] [Ideas] [answered]
The scripts, especially render.js should allow for translations.  Right now, people have to dive-in render.js and edit the code to translate every piece in English to their own language. IMO that's okay for the HTML code, but not for the basic functions of the website.

**Comment by gabriel4649** [comment]
I can work on this task. I was thinking of using [i18next](https://github.com/jamuhl/i18next) for the task. What are your thoughts on using that library for this task? Did you have a particular solution in mind? Are there any guidelines I should follow for this task. Thanks for your great work on tosdr!

**Comment by michielbdejong** [comment]
@climatewarrior great! basically only two things to keep in mind:
- document how the build instructions change (if at all)
- document how people can add a language (i.e. point to whatever file they should add/change to build the site in a different language).

the current build process is documented in the README file, it's basically "sh build.sh" :)

**Comment by gabriel4649** [comment]
@michielbdejong Great! I will start working on it soon. I will submit a pull request later on.

**Comment by hugoroy** [comment]
@tatsuhirosatou hello, any news about this? We're going to work on this maybe for tosdr.org/conf -- it would be nice if you could join on irc.freenode.net #tosdr

**Comment by JustinBack** [comment, marked as answer]
Translations are now possible via Crisp!

https://github.com/JustinBack/CrispCMS-ToS-DR/tree/master/translations
